src/dotnet/Core/Channels/AsyncEnumerableExt.cs

```diff
@@ -138,6 +138,31 @@ await foreach (var item in source.WithCancellation(cancellationToken).ConfigureA
         }
     }
 
+    public static async IAsyncEnumerable<List<TSource>> Buffer<TSource>(
```

Comment:
I am pretty sure the method is called `Chunk` in .NET now + maybe it exists for AE in Fusion.

Reply:
I can rename it to `Chunk`